### DISCUSSION TITLE
Sharing: Remote (No Internet) Frigate environment

### DISCUSSION THREAD

**Discussion by kbrouder** [blakeblackshear/frigate#3319] [Show and tell]
Just wanted to share in case anyone has a similar need. I made this to serve security cameras at a remote spot that doesn't have an internet connection currently, will be running 1-2 cameras. The hardware is a Rock 3A + Coral Dual Edge TPU (E Key only getting 1), POE Switch, 4G LTE Modem, 2 TB hdd, 12v PSU, external 4G antenna. It is running Frigate, eclipse-mosquitto, hass (docker), and remote access through cloudflare tunnels. 

Could probably do this without hass, but I didnt want to make any custom software, so the hass instance is purely just to grab the mqtt notifications and send push notifications via pushover. Well see how well this holds up to the elements with that fan, I decided I better give it some external airflow...I will mount it somewhat shielded from the rain....

If I were to do it again I would probably try to find a POE switch I could have powered from the PSU, that was my original plan, but it didn't pan out, this one is pretty new so it needed a lot more than the 50w PSU could provide, so I had to hack up the psu that came with it and hide it in there. I bricked one of my Rock 3As trying to power it from the PSU, so I decided to run that off the POE switch as well. 

Happy to provide info or answer any questions. 

![20220608_205625](https://user-images.githubusercontent.com/6267127/172743036-d39e5c2b-879a-4720-a40c-4721bbb717bf.jpg)

**Comment by Cold-Lemonade** [comment]
Very cool. How is this working? How much are you paying for your internet connection over the mobile network?
